src/window.cpp

```diff
@@ -38,6 +38,7 @@ const QVersionNumber version = QVersionNumber::fromString("0.1.0");
 Window::Window()
 {
     trayIconIcon = new QIcon(":/resources/images/minikube.png");
+    trayIconGrayIcon = new QIcon(":/resources/images/minikube-disabled.png");
```

Comment:
Since this is only used by the tray, I suggest we move creating this icon to the tray constructor. The only reason `trayIconIcon` is here is because we reuse the icon for other things .